font/src/ft/mod.rs

```diff
@@ -265,13 +267,35 @@ impl FreeTypeRasterizer {
                 Some(FixedSize { pixelsize: pixelsize.next().expect("has 1+ pixelsize") })
             };
 
+            // Works fine btw.
+            let mut load_flags = Self::ft_load_flags(pattern);
+            let has_color = ft_face.has_color();
+            if has_color {
+                unsafe {
+                    let ft_face = ft_face.raw_mut();
+                    let best = 0;
+                    // XXX we could check sizes info, but we don't know to what size we're going to
+                    // downscale.
+                    //
+                    // let available_sizes =
+                    // std::slice::from_raw_parts::<freetype_sys::FT_Bitmap_Size>(
```

Comment:
We could try and check bitmap sizes and pick the most close one to our current cell dimensions, however pass a size information here requires a lot of api changing.